kubernetes-agent/values.yaml

```diff
@@ -1,13 +1,29 @@
 image:
-  repository: 'public.ecr.aws/f2p6e0z8/kubernetes-agent'
+  repository: 'kubernetes-agent'
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: ''
+  tag: 'collector'
+
+sbomCollector:
+  repository: 'kubernetes-sbom-collector'
+  pullPolicy: IfNotPresent
+  tag: 'collector'
+  name: 'aikido-kubernetes-sbom-collector'
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+      ephemeral-storage: 500Mi
+    limits:
+      cpu: 500m
+      memory: 1Gi
+      ephemeral-storage: 1Gi
 
 config:
   apiToken: ''
   apiEndpoint: 'https://k8s.aikido-security.com'
   controllerCacheSyncTimeout: "5m"
+  isSBOMCollectionEnabled: false
```

Comment:
Let's change the semantics of this flag to determine whether the SBOM collector is installed as a DaemonSet or as a Deployment